### PR TITLE
Features/styled components

### PR DESCRIPTION
**Pull request by waleedbinalim**
added styled components

### CODE REVIEW

File: src/components/Orders.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+export default function Orders() {
+    const {filteredOrders , setFilteredOrders , orders , setOrders , orderSearch , setOrderSearch,
+    openEditModal, setOpenEditModal , editOrderValue , setEditOrderValue} = useContext(OrderContext)
+
+
+    useEffect(() => {
+        fetch('http://localhost:5000/orders/all')
+        .then(res => res.json())
+        .then(data => {
+            setOrders(data)
```
Comment (zain5555) [inline src/components/Orders.js]: put this string in constants and use it here then.

File: src/components/Orders.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    let handleDelete= async (e) => {
+        const deleteURL = `/orders/${e.currentTarget.id}` ;
+        await api.put(deleteURL, {isActive: false} , {headers: headers})
+        let updatedOrders = await api.get('/orders/all')
+        setFilteredOrders(updatedOrders.data)
+        setOrders(updatedOrders.data)
+    }
+
+    let handleEdit= async (e) => {
+        console.log(e.currentTarget.id)
```
Comment (zain5555) [inline src/components/Orders.js]: remove consoles always, check for them vigilantly.

File: src/components/Users.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    }
+`;
+
+
+export default function Users() {
+    const {filteredUsers, setFilteredUsers , users , setUsers , userSearch , setUserSearch,
+        openEditModal , setOpenEditModal , editUserValue , setEditUserValue} = useContext(UserContext)
+
+    useEffect( ()=> {
+        fetch('http://localhost:5000/users/all')
```
Comment (zain5555) [inline src/components/Users.js]: string should be api call

File: src/components/users/UserCards.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                                                id={user._id} 
+                                                variant="contained" size="small" color="primary" 
+                                                onClick={(e) => {handleEdit(e)}}
+                                            >
+                                            EDIT
+                                            </Button>
+                                            <Button 
+                                                id={user._id} 
+                                                variant="contained" 
+                                                size="small"
```
Comment (zain5555) [inline src/components/users/UserCards.js]: Can we further make this return statement into one component and use that here.

**Review by zain5555 — CHANGES_REQUESTED** [review]
Pr reviewed

File: src/components/users/UserCards.js (hunk 4 above)
Comment (zain5555) [reply src/components/users/UserCards.js]: @waleedbinalim please resolve this as well so I can merge it.

Comment (waleedbinalim) [reply src/components/users/UserCards.js]: Trying to figure how to set ID's on it when the button groups are in a separate component

Comment (waleedbinalim) [reply src/components/users/UserCards.js]: should I do the same for order cards as well?

**Pull request merged**